doc/tutorial/staramr-tutorial.ipynb

```diff
@@ -370,35 +433,148 @@
     "cut -f 1,2,5,6,7,8,10 out/pointfinder.tsv | column -s$'\\t' -t"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Isolate ID       Gene   %Identity  %Overlap  HSP Length/Total Length  Contig                  Start  End    Accession\n",
+      "GCF_001478105.1  IncI1  100.00     100.00    142/142                  ref|NZ_JYVD01000056.1|  15896  16037  AP005147\n",
+      "GCF_001478105.1  IncX1  100.00     100.00    373/373                  ref|NZ_JYVD01000049.1|  2546   2174   CP001123\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Show all columns\n",
+    "column -s$'\\t' -t out/plasmidfinder.tsv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Isolate ID       Gene   %Overlap  HSP Length/Total Length  Contig\n",
+      "GCF_001478105.1  IncI1  100.00    142/142                  ref|NZ_JYVD01000056.1|\n",
+      "GCF_001478105.1  IncX1  100.00    373/373                  ref|NZ_JYVD01000049.1|\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Show a subset of columns\n",
+    "cut -f 1,2,4,5,6 out/plasmidfinder.tsv | column -s$'\\t' -t"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This shows all the aquired point mutations leading to antimicrobial resistance, one per line."
+    "This shows all the aquired point mutations and plasmids genes leading to antimicrobial resistance, one per line."
```

Comment:
Could you move this sentence in between showing the `pointfinder.tsv` results and `plasmidfinder.tsv` results? Then, add another sentence after `plasmidfinder.tsv` results saying something like `This shows all the plasmid genes detected in the genomes.`.